go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/caarlos0/env/v6 v6.2.2
+	github.com/go-logr/logr v0.1.0
```

Comment:
It was used, but it wasn't referenced directly from the project files, but from vendor pkgs. Anyway the same change exists in the predefined name PR, so it will disappear from here